jobs/storage_test.groovy

```diff
@@ -0,0 +1,57 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = "storage_backend"
+repoName = 'charts'
+
+job(name) {
+  description """
+    <p>Runs the <a href="https://github.com/deis/workflow-e2e">e2e tests</a> against a <a href="https://github.com/deis/charts/tree/master/${defaults.workflowChart}">${defaults.workflowChart}</a> chart</p>
```

Comment:
maybe just add a mention about the particular use of this job, which is to run existing e2e tests but using a non-default storage backend...